lib/commands/nw-test/index.js

```diff
@@ -34,6 +36,37 @@ module.exports = {
     this.quickTemp.remove(this, '-testsDistNW');
   },
 
+  TestServerBuilder: Builder.extend({
```

Comment:
> this will impact `ember build`.

I think this is fine, because we're only tweaking the build if the environment being built for is `test`.

> However a slight functional problem that I can't seem to figure out is how to hide or show nwjs.

I'm thinking we can just do away with `window.show`. Hiding it isn't important so long as NW.js doesn't stay running in CI mode. For instance, when you configure Testem to run Chrome or Firefox in CI, those browsers will be visible for the duration of the tests, then close immediately afterwards.